human_experiments/scripts/run_minecraft_metadata_validation.py

```diff
@@ -0,0 +1,92 @@
+import sys 
```

Comment:
Add a docstring for this script at the top explaining what it does

Reply:
@CalebUAz  - Is this comment truly resolved? I do not see the docstring at the top of the script.